### ISSUE TITLE
Cooldowns no longer in about:config?

### ISSUE THREAD

**Issue by FishHeadswg** [ihavenoface/4chan-x#279]
I've always had problems with cooldowns and I would just go into about:config to reset them, but as of this morning they're mysteriously missing and I can't track down where they're being stored.

What's going on here? Was something changed?

**Comment by ihavenoface** [comment]
Greasemonkey changed the way it stores the settings. It's in some db file located in gm_scripts which is in your Firefox profile.
To modify the file you can go to about:support. There should be a button near `Profile Folder`.

**Comment by FishHeadswg** [comment]
Figures. I actually found that file before opening this issue and deleted everything in cooldown.x and nothing happened. I figured it was just some left over vestigial file.

After looking through it again on your suggestion apparently there's a second cooldown.x. Wiping that seems to have reset it.

Editing GS data has suddenly become a massive pain. I wish they just left it alone.

**Comment by ihavenoface** [comment]
I found it to be odd as well. But not trashing up firefox's main config, with leftovers from uninstalled scripts for instance, is probably not a bad idea.

**Comment by FishHeadswg** [comment]
Yep. I also take back what I said. I wiped the cooldown info but the cooldown somehow magically restored itself. 

This is suddenly becoming a real pain in the butt.

**Comment by ihavenoface** [comment]
Most likely because it's still in memory and then rewritten by the script.
What exactly is wrong with the cooldown anyway? The way the configuration is stored surely isn't the root of the problem.

**Comment by FishHeadswg** [comment]
Well, I went into SQLite Manager instead of being lazy and using Notepad++ and edited it again. Seems to have stuck this time.

Also, nothing is wrong with the cooldown function in the actual script. I just have a problem with a certain board's new ridiculously long cooldowns and it ends up being faster to just reset my IP than to wait for the cooldown, so that's what I do.

**Comment by ihavenoface** [comment]
I see.
I'll probably add some hotkey that will clear the cooldown when needed, so please leave the issue open.

**Comment by FishHeadswg** [comment]
Reopened.
